scaffolder-templates/roadie-agent-scaffolder-action/skeleton/src/index.js

```diff
@@ -0,0 +1,18 @@
+import { RoadieAgent, createRoadieAgentScaffolderAction } from '@roadiehq/roadie-agent';
+import { myScaffolderActionHandler } from './myScaffolderActionHandler.js';
+
+RoadieAgent.fromConfig({
+    server: 'https://${{ values.tenant }}.broker.roadie.so',
+    port: 7342,
```

Comment:
I've modified the agent that both ports can now be omitted and they default to these values